core/lib/server/pub.ts

```diff
@@ -1358,15 +1359,20 @@ interface GetPubsWithRelatedValuesAndChildrenOptions extends GetManyParams, Mayb
 type PubIdOrPubTypeIdOrStageIdOrCommunityId =
 	| {
 			pubId: PubsId;
+			pubIds?: never;
 			pubTypeId?: never;
 			stageId?: never;
 			communityId: CommunitiesId;
 			userId?: UsersId;
 	  }
 	| {
 			pubId?: never;
-			pubTypeId?: PubTypesId;
-			stageId?: StagesId;
+			/**
+			 * Multiple pubIds to filter by
+			 */
+			pubIds?: PubsId[];
+			pubTypeId?: PubTypesId[];
+			stageId?: StageConstraint[];
```

Comment:
not reeeaally happy with having `pubTypeId` and `stageId` be singulars that accept arrays. mostly i did not want to introduce a bunch of noise to this PR, but i guess i already did

should i
a. change `pubTypeId`->`pubTypeIds`, same for stages
b. have both `pubTypeId?: PubTypesId, pubTypeIds?: PubTypesIds[]`
c. keep it like this


i think having `pubIds` separate from `pubId` makes sense, as they indicate a difference in the returned type (`pubId` returns a singular pub, while `pubTypeIds` returns an array)